index.bs

```diff
@@ -839,6 +838,21 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
                     bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
 
+                1.  Let |attestationPreference| be the value of |options|.{{MakePublicKeyCredentialOptions/attestation}}.
+                    <dl class="switch">
+                    <dt>If the value of |attestationPreference| is "none" (or it has been left unspecified),</dt>
+                    <dd>replace potentially identifying information (such as AAGUID and attestation certificates) 
+                        in the attested credential data and attestation statement with non-identifying versions of the same data.</dd>
+
+                    <dt>If the value of |attestationPreference| is "verifiable",</dt>
+                    <dd>potentially replace the attestation statement with a more privacy-friendly and/or more easily verifiable
+                        version of the same data (for example, by employing a Privacy CA).</dd>
```

Comment:
use [=Privacy CA=]

the distinction here from "none" is that in case the RP ends up needing it, there is possibly support for retroactive attestation unblinding, yes?

Reply:
The "obtain user consent" handling should be identical for "verifiable" and "direct".  In both cases the authenticator model will be known to some other party (either the Privacy CA or the RP).
In my opinion Authenticator models satisfying the FIDO requirements (i.e. batch size >= 100000 and others) do not lead to a privacy risk and hence do not require yet another user consent (in addition to the general consent of using the authenticator a creating a credential).